apps/meteor/app/cloud/server/functions/saveRegistrationData.ts

```diff
@@ -32,9 +29,29 @@ export function saveRegistrationData({
 		Settings.updateValueById('Cloud_Workspace_Client_Secret_Expires_At', client_secret_expires_at),
 		Settings.updateValueById('Cloud_Workspace_PublicKey', publicKey),
 		Settings.updateValueById('Cloud_Workspace_Registration_Client_Uri', registration_client_uri),
-		Settings.updateValueById('Cloud_Workspace_License', licenseData.license || ''),
 	]).then(async (...results) => {
-		await callbacks.run('workspaceLicenseChanged', licenseData.license);
+		// wait until all the settings are updated before syncing the data
```

Comment:
I guess this is mostly for multi-instance setups to receive and cache the settings?

Reply:
the callback is the function that applies the license, the function does the same internally and the other instances replicate through the setting